claimtxman/claimtxman.go

```diff
@@ -418,7 +419,7 @@ func (tm *ClaimTxManager) ReviewMonitoredTx(ctx context.Context, mTx *ctmtypes.M
 		Data:  mTx.Data,
 	})
 	if err != nil {
-		err := fmt.Errorf("failed to estimate gas: %w", err)
+		err := fmt.Errorf("failed to estimate gas: %v", err)
```

Comment:
Is this any different? If I'm not mistaken, the `%w` is to support `Unwrap` of the returned error, right? It shouldn't change how it's sent to the logs.

Reply:
Depending on how the error is created, %w can give problems. It is safer use %v which is the generic and will work in any case